client/js/src/types.ts

```diff
@@ -20,7 +20,7 @@ export interface Config {
 	show_api: boolean;
 	stylesheets: string[];
 	path: string;
-	protocol?: "sse_v2" | "sse_v1" | "sse" | "ws";
+	protocol?: "sse_v2.1" | "sse_v2" | "sse_v1" | "sse" | "ws";
```

Comment:
v2.1 changes the format of FileData responses: they now include a `meta` field to make them easier to identify